src/server/Utils/Datasets/E-n29-k4-s7-c200-ecap100.csv

```diff
@@ -0,0 +1,113 @@
+﻿ID;NODE_COORD_X;NODE_COORD_Y;DEMAND
```

Comment:
Some information are missing using this storage form. Where do you store it ?

Reply:
Good point. I will add the missing information asap.